File: 1.two sum/twosum.js
var twoSum = function (nums, target) {
    for (let i = 0; i < nums.length; i++) {
        const num1 = nums[i];
        const complement = target - num1;

        for (let j = i + 1; j < nums.length; j++) {
            const num2 = nums[j];

            if (num2 === complement) {
                return [i, j];
            }
        }
    }
    return [];
};

File: 1.two sum/twosum.c
#include <stdlib.h>
#include <stdio.h>>

int *twoSum(int *nums, int numsSize, int target, int *returnSize)
{
    int *numIndices = (int *)malloc(sizeof(int) * numsSize);
    int complement;
    int i;

    for (i = 0; i < numsSize; i++)
    {
        complement = target - nums[i];
        for (int j = 0; j < i; j++)
        {
            if (nums[j] == complement)
            {
                numIndices[0] = j;
                numIndices[1] = i;
                *returnSize = 2;
                return numIndices;
            }
        }
    }

    *returnSize = 0;
    return NULL;
}